### ISSUE TITLE
Lost all configurasions and ssl certs in web dashboard

### ISSUE THREAD

**Issue by mattiasghodsian** [NginxProxyManager/nginx-proxy-manager#2882]
"NPM" just whent back to default and i lost everything when i pulled the latest docker image.My old configs are still ruining and working, don't understand.....

![image](https://user-images.githubusercontent.com/20265517/236007780-a522f3b9-e74c-42cf-9d2d-ac3610498664.png)

Is it possible to re import them ? 
```bash
~/docker/app/nginxproxymanager/data/nginx/proxy_host$ ls
10.conf  14.conf  17.conf  19.conf  24.conf  27.conf  29.conf  32.conf  34.conf  5.conf
13.conf  16.conf  18.conf  2.conf   26.conf  28.conf  31.conf  33.conf  4.conf   7.conf
```

**Comment by smcoder0707** [comment]
The same thing happened to me and when I checked the logs I found that it was using SQLite database while I had MySQL configured in the JSON file and as an environment variable.
For now I switched to old 2.9.22 version to quickly get rid of this issue. will recheck later when this is fixed.

**Comment by DravenTec** [comment]
I don't think this is really a bug. In the release notes of 2.9.22 it said the following:

The next release v2.10.0 will have changes that may adversely affect you, please pin your docker tag to 2.9.22 and upgrade after you conduct a full backup.

**Comment by smcoder0707** [comment]
> I don't think this is really a bug. In the release notes of 2.9.22 it said the following:
> 
> * The next release v2.10.0 will have changes that may adversely affect you, please pin your docker tag to 2.9.22 and upgrade after you conduct a full backup.
> 
> I had the same problem and only read the release notes carefully afterwards... shame on me.

The data is not really lost. with the new version, he added support for SQLite and also added support for providing configuration through environment variables.

So, after upgrading to this new version, nginx proxy manager is using SQLite database even  though MySQL is specified as the database backend. As there is no existing SQLite database, It provides blank interface with no previous settings.

TLDR; For some reason, it is not using MySQL as the database backend on this new version, even though it is explicitly specified in the configuration.

**Comment by mattiasghodsian** [comment]
> I don't think this is really a bug. In the release notes of 2.9.22 it said the following:
> 
> * The next release v2.10.0 will have changes that may adversely affect you, please pin your docker tag to 2.9.22 and upgrade after you conduct a full backup.
> 
> I had the same problem and only read the release notes carefully afterwards... shame on me.

I rolled-back to v2.9.22 for now
